api_docs/include/rest-endpoints.md

```diff
@@ -117,3 +117,4 @@
 
 * [Fetch an API key (production)](/api/fetch-api-key)
 * [Fetch an API key (development only)](/api/dev-fetch-api-key)
+* [Send test notification to mobile device](/api/test-notify)
```

Comment:
Nit: match the page's title, like the other specialty-endpoints pages

```suggestion
* [Send a test notification to mobile device](/api/test-notify)
```

or, following a suggestion elsewhere in my review:

```suggestion
* [Send a test notification to mobile device(s)](/api/test-notify)
```